site/content/docs/5.3/forms/checks-radios.md

```diff
@@ -422,45 +422,65 @@ Add `.star-rating-dark` to the `.star-rating` for a dark variant.
-    <label for="mixed3" title="Mixed"></label>
+    <label for="mixed3" title="Mixed" aria-label="Mixed"></label>
 
     <input type="radio" id="good3" name="rating" value="4" class="visually-hidden" checked>
-    <label for="good3" title="Good"></label>
+    <label for="good3" title="Good" aria-label="Good"></label>
 
     <input type="radio" id="excellent3" name="rating" value="5" class="visually-hidden">
-    <label for="excellent3" title="Excellent"></label>
+    <label for="excellent3" title="Excellent" aria-label="Excellent"></label>
   </fieldset>
 </form>
 {{< /example >}}
 
+### Readonly
+Make star ratings readable but non editable by using `<span>` instead of `<input>` elements.
+
+{{< example >}}
+<div class="star-rating">
+  <p class="visually-hidden">Results relevance: rated 3 stars out of 5</p>
+
+    <div aria-hidden="true">
+      <span id="terrible4" class="checked"></span>
+      <span id="bad4" class="checked"></span>
+      <span id="mixed4" class="checked"></span>
+      <span id="good4"></span>
+      <span id="excellent4"></span>
+    </div>
+
+</div>
+{{< /example >}}
+
 ### Disabled
 
 Make star ratings look inactive inside or outside a form by adding the `disabled` boolean attribute to the `<fieldset>` element and the `checked` boolean attribute to any `<input>` element.
 
 {{< example >}}
-<fieldset class="star-rating" disabled>
-  <input type="radio" id="terrible4" name="rating" value="1" class="visually-hidden">
-  <label for="terrible4" title="Terrible"></label>
+<fieldset class="star-rating" disabled aria-hidden="true">
+  <legend class="visually-hidden">Results relevance</legend>
+
+  <input type="radio" id="terrible5" name="rating" value="1" class="visually-hidden">
+  <label for="terrible5" title="Terrible"></label>
 
-  <input type="radio" id="bad4" name="rating" value="2" class="visually-hidden">
-  <label for="bad4" title="Bad"></label>
+  <input type="radio" id="bad5" name="rating" value="2" class="visually-hidden">
+  <label for="bad5" title="Bad"></label>
 
-  <input type="radio" id="mixed4" name="rating" value="3" class="visually-hidden" checked>
-  <label for="mixed4" title="Mixed"></label>
+  <input type="radio" id="mixed5" name="rating" value="3" class="visually-hidden" checked>
+  <label for="mixed5" title="Mixed"></label>
 
-  <input type="radio" id="good4" name="rating" value="4" class="visually-hidden">
-  <label for="good4" title="Good"></label>
+  <input type="radio" id="good5" name="rating" value="4" class="visually-hidden">
+  <label for="good5" title="Good"></label>
 
-  <input type="radio" id="excellent4" name="rating" value="5" class="visually-hidden">
-  <label for="excellent4" title="Excellent"></label>
+  <input type="radio" id="excellent5" name="rating" value="5" class="visually-hidden">
+  <label for="excellent5" title="Excellent"></label>
 </fieldset>
-<span class="visually-hidden">rated 3 stars out of 5</span>
+<p class="visually-hidden">Results relevance: rated 3 stars out of 5</p>
```

Comment:
Why use this more complicated method? 
Isn't it ok to remove `aria-hidden="true"` and let screen reader vocalize the radio buttons?
@Aniort an opinion on this?

Reply:
@hannah, we have to take time during a Web sp$ec review so i can explain to you why we choose that method!

Reply:
@Aniort ok thanks :-)